docs/submitting.md

```diff
@@ -1,35 +1,41 @@
 Submitting a paper to JOSS
 ==========================
 
-If you've already licensed your code and have good documentation then we expect that it should take less than an hour to prepare and submit your paper to JOSS.
+If you've already developed a fully featured research code, released it under an OSI-approved license, and written good documentation and tests, then we expect that it should take perhaps an hour or two to prepare and submit your paper to JOSS.
+But please read these instructions carefully for a streamlined submission.
 
 ## Typical paper submission flow
 
-Before you submit we need you to:
+Before you submit, you should:
 
-- Make software available in an open repository (GitHub, Bitbucket etc.) and include an [OSI approved open source license](https://opensource.org/licenses).
-- Make sure that the software complies with the [JOSS review criteria](review_criteria.html).
-- Author a short Markdown paper `paper.md` with a title, summary, author names, affiliations, and key references. See an example [here](https://github.com/arfon/fidgit/tree/master/paper).
```

Comment:
why not link the URL to "example"?

Reply:
I missed that one

Reply:
That's deleted text. Edited version does link on "example paper."